### PR TITLE
Remove container and button gradient default values

### PR DESCRIPTION
**Pull request by JeanPaiva**
This removes gradient default values for container and button blocks.

### CODE REVIEW

File: includes/class-legacy-attributes.php [hunk 1]
```diff
@@ -72,6 +84,23 @@ public static function get_settings( $version, $block, $settings, $atts ) {
 				$settings['paddingRight'] = $legacy_settings['paddingRight'];
 				$settings['paddingBottom'] = $legacy_settings['paddingBottom'];
 				$settings['paddingLeft'] = $legacy_settings['paddingLeft'];
+
```
Comment (tomusborne) [inline includes/class-legacy-attributes.php:87]: Missing `gradientDirection`

File: includes/class-legacy-attributes.php [hunk 2]
```diff
@@ -72,6 +84,23 @@ public static function get_settings( $version, $block, $settings, $atts ) {
...
+				$settings['gradientColorTwo'] = $legacy_settings['gradientColorTwo'];
+				$settings['gradientColorTwoOpacity'] = $legacy_settings['gradientColorTwoOpacity'];
+			}
+
+			if ( 'button' === $block ) {
+				$button_legacy_settings = wp_parse_args(
+					$atts,
+					$legacy_defaults['button']
+				);
+
```
Comment (tomusborne) [inline includes/class-legacy-attributes.php:99]: Missing `gradientDirection`

**Review by tomusborne — COMMENTED** [review]
Also missing the code that generates the legacy CSS on the front-end for buttons. For example: https://github.com/tomusborne/generateblocks/blob/try-default-removal/includes/generate-css.php#L86-L91

**Review by tomusborne — APPROVED** [review]
This looks good. We should keep in mind that we're using `hasNumericValue()` for non-numeric values (in other places too). We should probably add a `hasValue()` util soon.

**Comment by JeanPaiva** [comment]
The function itself is a value check we can rename all occurrences to `hasValue`. In the future, we may want to add a `getNumberValue` function which parses the value to an integer, check if is NaN, and return the formatted value.

**Pull request merged**